create-page/create.js

```diff
@@ -0,0 +1,20 @@
+import { createItem, checkAuth } from '../fetch-utils.js';
+
+const createForm = document.getElementById('create-form');
+
+createForm.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    const data = new FormData(createForm);
+    const newItem = {
+        name: data.get('name'),
+        qty: data.get('qty'),
+    };
+
+    const response = await createItem(newItem);
+    console.log(response);
+
+    location.replace('/');
```

Comment:
this currently works, because you have a redirect function that's automatically redirecting the user from auth to the list - but if you wanted to avoid that page hopping, you could update this to route to your list page instead